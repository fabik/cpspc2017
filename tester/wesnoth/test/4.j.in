90000 51
1
45901
