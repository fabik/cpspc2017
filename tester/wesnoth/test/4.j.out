24 47 0.5548237237
